final_prep/mod_zero_hero.rb

```diff
@@ -1,52 +1,113 @@
 # Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 # Declare two variables - hero_name AND special_ability - set to strings
+hero_name = "Hellboy"
+special_ability = "superhuman strength"
 
 # Declare two variables - greeting AND catchphrase
 #   greeting should be assigned to a string that uses interpolation to include the hero_name
 #   catchphrase should be assigned to a string that uses interpolation to include the special_ability
+greeting = "Now introducing #{hero_name}!"
+puts greeting
+catchphrase = "Among immunity to fire, he also has #{special_ability}."
+puts catchphrase
 
 # Declare two variables - power AND energy - set to integers
+power = 100
+energy = 200
 
 # Declare two variables - full_power AND full_energy
 #   full_power should multiply your current power by 500
 #   full_energy should add 150 to your current energy
+full_power = power * 500
+full_energy = energy + 150
+puts full_power
+puts full_energy
 
 # Declare two variables - is_human and identity_concealed - assigned to booleans
-
+is_human = false
+identity_concealed = false
 
 # Declare two variables - arch_enemies AND sidekicks
 #   arch_enemies should be an array of at least 3 different enemy strings
 #   sidekicks should be an array of at least 3 different sidekick strings
+arch_enemies = ["Rasputin", "Hecate", "Baba Yaga"]
+sidekicks = ["Abe Sapien", "Sarah Hughes", "Azzael"]
 
 # Print the first sidekick to your terminal
-
+puts sidekicks[0]
 # Print the last arch_enemy to the terminal
-
+puts arch_enemies[-1]
```

Comment:
Nice strategy- this will always give you the last element even as the contents of the array change.